Update tiflash-configuration.md (#18828)
pingcap/docs: tiflash/tiflash-configuration.md

    ## see known issue [#5576](https://github.com/pingcap/tiflash/issues/5576).
    # dt_enable_logical_split = false

    ## `max_threads`: The default value is the number of CPU cores. This parameter
    ## is the maximum number of threads executed. This parameter only takes effect
    ## when the system variable `tidb_max_tiflash_threads` is set to -1.

> 

Reviewer: ```suggestion
    ## For example, if the number of CPU cores is 32, you can set the parameter as follows:
    max_threads = 32
    
```

    ## see known issue [#5576](https://github.com/pingcap/tiflash/issues/5576).
    # dt_enable_logical_split = false

>     ## `max_threads`: The default value is the number of CPU cores. This parameter

Reviewer 2: The default value is `0`, and when it is set to 0, TiFlash will use the number of cpu cores as the execute concurrency.

>     ## is the maximum number of threads executed. This parameter only takes effect

Reviewer 2: This parameter is the internal execute concurrency when TiFlash execute a MPP task.

    ## see known issue [#5576](https://github.com/pingcap/tiflash/issues/5576).
    # dt_enable_logical_split = false

>     ## `max_threads`: The internal execution concurrency when TiFlash executes an MPP task.

Reviewer 3: ```suggestion
    ## `max_threads` indicates the internal thread concurrency when TiFlash executes an MPP task.
```

>     ## `max_threads`: The internal execute concurrency when TiFlash execute a MPP task.
>     ## The default value is 0 and when it is set to 0,
>     ## TiFlash will use the number of cpu cores as the execute concurrency.
>     ## This parameter only takes effect
>     ## when the system variable `tidb_max_tiflash_threads` is set to -1.
>     max_threads = 0

Reviewer: ```suggestion
    ## `max_threads`: The internal execution concurrency when TiFlash executes an MPP task.
    ## The default value is 0. When it is set to 0,
    ## TiFlash uses the number of CPU cores as the execution concurrency.
    ## This parameter only takes effect
    ## when the system variable `tidb_max_tiflash_threads` is set to -1.
    max_threads = 0
```
